recipes/sobjectizer/all/conanfile.py

```diff
@@ -84,9 +84,14 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
 
     def package_info(self):
-        self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.names["cmake_find_package"] = "sobjectizer"
+        self.cpp_info.names["cmake_find_package_multi"] = "sobjectizer"
+        cmake_target = "SharedLib" if self.options.shared else "StaticLib"
```

Comment:
https://github.com/Stiffstream/sobjectizer/blob/73b0d1bf5b4bdf543c24bc4969d2c408e0cea812/dev/so_5/CMakeLists.txt#L110 👍

Reply:
Well this line is for `add_subdirectory()`.

For imported targets, you must check these lines:
https://github.com/Stiffstream/sobjectizer/blob/73b0d1bf5b4bdf543c24bc4969d2c408e0cea812/dev/so_5/CMakeLists.txt#L111
https://github.com/Stiffstream/sobjectizer/blob/73b0d1bf5b4bdf543c24bc4969d2c408e0cea812/dev/so_5/CMakeLists.txt#L147
https://github.com/Stiffstream/sobjectizer/blob/73b0d1bf5b4bdf543c24bc4969d2c408e0cea812/dev/so_5/CMakeLists.txt#L178